src/api/app/controllers/webui/project_controller.rb

```diff
@@ -106,6 +106,7 @@ def update
     respond_to do |format|
       format.js do
         if @project.update(project_params)
+          @project.store
```

Comment:
Is this a separated bugfix that came up along, or it's part of the current implementation?

Reply:
A separated buggix. The description was only saved in the database table, but not in the `_meta` in the backend.

Retrieving a project meta from the API after changing the desciption with the web-UI didn't reflect the changes.

Reply:
> On top of that, I wonder why it is not true for [package_update](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/package_controller.rb#L118) then

The package model has an `after_save :write_to_backend`. The project model doesn't have it. See: https://github.com/openSUSE/open-build-service/blob/13c6c05913c849624e8e22f3e2ca3c17e2459830/src/api/app/models/package.rb#L74